### PR TITLE
Raredesc

### PR DESCRIPTION
**Pull request by Incoming5643**
<a href = "https://www.youtube.com/watch?v=wKxw_NwF5pE">341s6</href>

### CODE REVIEW

**Comment by optimumtact** [comment]
I can't believe you've done this

**Comment by RemieRichards** [comment]
>All these Smashmouth references

# GOD WHY

**Comment by bgobandit** [comment]
meme pr

**Comment by RemieRichards** [comment]
Suggestion:
Worm hole projector gun, Raredesc = "wubbalubbadubdub"

**Comment by Incoming5643** [comment]
Lucidity has been regained. It has been a fun 20 minutes.

**Comment by Cuboos** [comment]
NO, MERGE, I WANT THIS

**Comment by Incoming5643** [comment]
FORGE YOUR OWN DESTINY CUBOOS, FAR FROM THIS MEME LAND.

**Comment by Cuboos** [comment]
I NEED MY MEMES MAN... I GOTTA HAVE MY MEMES, DON'T UNDERSTAND!

**Comment by Miauw62** [comment]
please keep this open for april first

**Comment by Anonus** [comment]
THE MEMES HAVE BEGUN

**Comment by MrPerson** [comment]
I actually think there's a way to do this without being quite so.... annoying.

I've long wanted a crunch description, a flavor description, and maybe a joke description. As long as the three are kept separate, I have no problem with some jokes here and there.

**Comment by Supermichael777** [comment]
make these the descriptions if your high.

File: code/game/atoms.dm [hunk 1]
```diff
@@ -230,7 +231,11 @@ its easier to just keep the beam vertical.
 	user << "\icon[src] That's [f_name]"
 
 	if(desc)
-		user << desc
+		if(raredesc && prob(0.1))
```
Comment (Iamgoofball) [inline code/game/atoms.dm]: prob accepts percents???

holy shit

Comment (Incoming5643) [inline code/game/atoms.dm]: prob(prob(1)*10))

**Comment by MrStonedOne** [comment]
**GOD DAMN IT!** NOW I GOT THIS SONG STUCK IN MY HEAD!

ALSO THE DIFF DOESN'T MATCH UP WITH THE ORDER OF THE LYRICS AND THAT BOTHERS ME MORE THEN IT SHOULD!

**Comment by xxalpha** [comment]
:+1:

**Comment by tuypo1** [comment]
I think instead of a random chance it should have something that triggers it maybe some fancy goggles or something

**Comment by tuypo1** [comment]
although now that I realised its just song lyrics I don't know what sort of item you would use

oh a shreck mask

**Comment by veganzombeh** [comment]
I would say lower the chance to something like 1/500, but the user has to be within 2 tiles or something.

**Comment by Aranclanos** [comment]
thanks for your contribution

**Comment by Anonus** [comment]
ARAN RUINS MEMES FOREVER

**Comment by Incoming5643** [comment]
To be clear there IS a real idea here for subjective descriptions for items.

But yes, filling the codebase with smashmouth lyrics is probably not the right way. :angel:

**Comment by Iamgoofball** [comment]
will revive

**Pull request closed without merging**